src/libraries/Microsoft.PowerFx.Json/Functions/JsonFunctionImpl.cs

```diff
@@ -19,6 +19,7 @@
 using Microsoft.PowerFx.Core.Types.Enums;
 using Microsoft.PowerFx.Functions;
 using Microsoft.PowerFx.Types;
+using static Microsoft.PowerFx.Syntax.PrettyPrintVisitor;
```

Comment:
Curious: why do we need this?

Reply:
We dont. Auto added by VS. Thanks for catching this.